libqtile/manager.py

```diff
@@ -1688,9 +1688,9 @@ def f(cmd):
             return
         mb.startInput(prompt, f, "qshell")
 
-    def cmd_addgroup(self, group, label=None):
```

Comment:
While I indeed prefer `name` than `group`, we should keep `group` so that we avoid breaking configs of people using the keyword. :man_shrugging:

Reply:
On the opposite, but for the same reason, we should keep `label` as the second argument!

Reply:
Oh that is sloppy work from me, sorry about that. Fixed and pushed again.